resources/github-actions.md

```diff
@@ -7,20 +7,70 @@ title: GitHub Actions
 ---
 
 # {{ page.title }}
+{:.no_toc}
 
 ## Overview
+{:.no_toc}
 
-Details on the self-hosted GitHub Actions runners provided by the RAPIDS Ops team.
+The RAPIDS team is in the process of migrating from Jenkins to GitHub Actions for CI/CD. The page below outlines some helpful information pertaining to the implementation of GitHub Actions provided by the RAPIDS Ops team. The official GitHub documentation for GitHub Actions is also useful and can be viewed [here](https://docs.github.com/en/actions).
 
 ### Intended audience
+{: .no_toc }
 
 Operations
 {: .label .label-purple}
 
 Developers
 {: .label .label-green}
 
-## Details
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+## Implementation
+
+The RAPIDS Ops team provides GPU enabled self-hosted runners for use with GitHub Actions to the RAPIDS and other select GitHub organizations.
+
+To ensure proper usage of these GPU enabled CI machines, the RAPIDS Ops team has adopted a strategy known as _Marking code as trusted by pushing upstream_ which is described in [this CircleCI blog post](https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/).
+
+The gist of the strategy is that the source code from trusted pull requests can be copied to a prefixed branch (e.g. `pull-request/<PR_NUMBER>`) within the source repository and CI can be configured to test only those prefixed branches rather than the pull requests themselves.
+
+Pull requests authored by members of the given GitHub organization are considered trusted and therefore are copied to a `pull-request/*` branch for testing automatically.
+
+Pull requests from authors outside of the GitHub organization must first be reviewed by a repository member with `write` permissions (or greater) to ensure that the code changes are legitimate and benign. That reviewer must leave an `/ok to test` (or `/okay to test`) comment on the pull request before it's code is copied to a `pull-request/*` branch for testing.
+
+The `/ok to test` comment is only valid for a single commit. Subsequent commits must be re-reviewed and validated with another `/ok to test` comment.
+
+### Ignoring Pull Request Branches in `git`
+
+One consequence of the strategy described above is that a lot of `pull-request/*` branches will be created and deleted in GitHub as pull requests are opened and closed. To avoid having these branches fetched locally, you can run the following `git config` command, where `upstream` in `remote.upstream.fetch` is the `git` remote name corresponding to the source repository:
+
+```sh
+git config \
+  --global \
+  --add "remote.upstream.fetch" \
+  '^refs/heads/pull-request/*'
+```
```

Comment:
We may need to add a note to specify that this requires having at least git version `2.29` or greater

Reply:
Done in 51e49bc.